### PR TITLE
add k8s to platform backup/restore

### LINKED ISSUE #7784: [Docs] update Platform backup to include Kubernetes
From @jvigil-yugabyte:

> Hey docs team -- can you help me to add a Kubernetes "tab" (as seen on other pages, like [this one](https://docs.yugabyte.com/latest/yugabyte-platform/install-yugabyte-platform/install-software/default/)) to the ["Back up and restore Yugabyte Platform" docs page](https://docs.yugabyte.com/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-yp) with instructions for how to backup/restore platform instances running on kubernetes?
>
> We are about to add support to the platform backup/restore script for platform instances running on remote Kubernetes installations and this feature needs some documentation. Here is a [WIP / rough draft of the page text](https://docs.google.com/document/d/1XqERp2LasneRoMsZTEd-ITLSgsQAG-bbzCrDi7X4Loc/edit#) that may be a good starting point.

### PR DESCRIPTION
**Pull request by polarweasel**
Resolves #7784

* Adds a k8s link to the default [backup/restore page](https://deploy-preview-7785--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-yp.md)
* Adds a [new k8s page](https://deploy-preview-7785--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md)

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+1. Verify that the machine performing the backup operation can access the Yugabyte Platform kubernetes pod instance:
+
+    ```sh
+    kubectl exec --namespace <kubernetes_namespace> -it <kubernetes_pod> -c yugaware -- cat /opt/yugabyte/yugaware/README.md
+    ```
+
+    <br/>
+
+    The Yugabyte Platform console is used in a highly available mode and orchestrates and manages YugabyteDB universes, or clusters, on one or more regions (across public cloud and private on-premises data centers). For more details, refer to the [Yugabyte Platform overview](https://docs.yugabyte.com/latest/yugabyte-platform/overview/).
```
Comment (jvigil-yugabyte) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md:54]: This didn't render correctly in the preview -- it was supposed to represent the output of running the above command (not a new paragraph). And, after looking at it again, it seems kind of confusing. Maybe we can just delete this line?

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+1. Copy the the Yugabyte Platform backup script (`yb_platform_backup.sh`) from the yugabyte-db repository to your local workstation using the following wget command:
+
+    ```sh
+    wget https://github.com/yugabyte/yugabyte-db/blob/master/managed/devops/bin/yb_platform_backup.sh
+    ```
+
+1. Verify that the machine performing the backup operation can access the Yugabyte Platform kubernetes pod instance:
+
+    ```sh
+    kubectl exec --namespace <kubernetes_namespace> -it <kubernetes_pod> -c yugaware -- cat /opt/yugabyte/yugaware/README.md
```
Comment (jvigil-yugabyte) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md:49]: Need to change 
* `<kubernetes_namespace>` -> `<k8s_namespace>`
* `<kubernetes_pod>` -> `<k8s_pod>`
to be consistent with the below command(s) and explanation(s).

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
```
Comment (jvigil-yugabyte) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md:97]: Same as above:
`<kubernetes_namespace>` -> `<k8s_namespace>`
`<kubernetes_pod>` -> `<k8s_pod>`

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    ```
+
+    <br/>
+
+    The Yugabyte Platform console is used in a highly available mode and orchestrates and manages YugabyteDB universes, or clusters, on one or more regions (across public cloud and private on-premises data centers). For more details, refer to the [Yugabyte Platform overview](https://docs.yugabyte.com/latest/yugabyte-platform/overview/).
```
Comment (jvigil-yugabyte) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md:102]: Same as above -- Maybe we can just delete this line?

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    * `exclude_releases` excludes Yugabyte releases from the backup archive.
+    * `verbose` prints debug output.
+
+    {{< note title="Prometheus metrics data" >}}
+
+Backup and restore of Prometheus metrics data is not currently supported.
+
+    {{< /note >}}
+
+1. Verify that the backup `.tar` file, with the correct timestamp, is in the specified output directory.
```
Comment (jvigil-yugabyte) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md:80]: Reword (.tar -> .tar.gz) (is in -> is created in):
1. Verify that the backup `.tar.gz` file, with the correct timestamp, is created in the specified output directory.

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+1. Run the `yb_platform_backup.sh` script using the `restore` command:
+
+    ```sh
+    ./yb_platform_backup.sh restore --input <input_path> --k8s_namespace <kubernetes_namespace> --k8s_pod <kubernetes_pod> [--verbose]
+    ```
+
+    <br/>
+
+    * `restore` is the command to restore the Yugabyte Platform server content.
+    * `input_path` is the path to the .tar backup file to restore.
```
Comment (jvigil-yugabyte) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md:118]: Reword (.tar backup file -> `.tar.gz` backup file):
* `input_path` is the path to the `.tar.gz` backup file to restore.

**Review by jvigil-yugabyte — CHANGES_REQUESTED** [review]
Noticed a few minor issues to fix. Thanks

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md (hunk 1 above)
Comment (polarweasel) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md:54]: Removed from all the procedures, and stuck it up top in the introduction...

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md (hunk 5 above)
Comment (polarweasel) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md:80]: Fixed here and in one other spot...

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md (hunk 1 above)
Comment (jvigil-yugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md]: :+1:

**Review by jvigil-yugabyte — APPROVED** [review]
Thanks, looks good!

**Pull request merged**